### PR TITLE
wine: install man page for wine executable

### LINKED ISSUE #43436: Missing man page for Wine
The `wine` package does not come with the man pages and there is no corresponding `wine-doc` package.

### PR DESCRIPTION
**Pull request by amak79**
#### Testing the changes
- I tested the changes in this PR: **briefly**
 Closes #43436

### CODE REVIEW

**Comment by amak79** [comment]
I need to make some changes so please do not merge yet.

**Comment by amak79** [comment]
I used `vinstall` to install the `wine.de.UTF-8.man`, `wine.fr.UTF-8.man` and `wine.pl.UTF-8.man` manuals. `vman` is unable to install them in the correct location because of the 'UTF-8' string in the filename, but this causes xlint to fail. Should I just rename the manuals before calling `vman`?

**Comment by classabbyamp** [comment]
I've fixed the underlying issue with `vman`  in #43686, but I think in this case it would make more sense to patch [this makefile](https://github.com/wine-mirror/wine/blob/wine-8.6/loader/Makefile.in) to do something more like [this makefile](https://github.com/wine-mirror/wine/blob/wine-8.6/server/Makefile.in#L47-L50)

**Comment by amak79** [comment]
I tried patching the makefile like you suggested but I couldn't get it to work. The auto-generated top-level makefile just doesn't have an install rule for the man pages. I ended up using `vman` with your fix applied. I will push my changes once your fix is merged.

**Comment by classabbyamp** [comment]
vman should work now

File: srcpkgs/wine/template [hunk 1]
```diff
@@ -112,6 +112,12 @@ post_install() {
 			ln -s ${_wine_libexec}/wine64 ${DESTDIR}/usr/bin/wine64
 		;;
 	esac
+
+	# install man page for wine executable
+	vman loader/wine.man wine.1
+	for lang in de fr pl ; do
+		vman loader/wine.${lang}.UTF-8.man wine.${lang}.UTF-8.1
```
Comment (classabbyamp) [inline srcpkgs/wine/template:119]:
```suggestion
	for f in loader/wine*.man; do
		vman "$f" "${f//man/1}"
```
i think this should work

Comment (amak79) [reply srcpkgs/wine/template]: I applied your suggestion. Thanks.

**Pull request merged**